app/dashboard/festivals/page.tsx

```diff
@@ -1,13 +1,24 @@
 import FestivalsTable from "@/app/components/organisms/festivals/festivals-table";
 import { fetchFestivals } from "@/app/data/festivals/actions";
+import Link from "next/link";
+import { Button } from "@/app/components/ui/button";
+import { PlusIcon } from "lucide-react";
 
 export default async function Page() {
   const festivals = await fetchFestivals();
 
   return (
-    <div className="container p-4 md:p6">
-      <h1 className="mb-2 text-2xl font-bold md:text-3xl">Festivales</h1>
+    <div className="container p-4 md:p-6">
+      <div className="flex items-center justify-between mb-4">
+        <h1 className="text-2xl font-bold md:text-3xl">Festivales</h1>
+        <Link href="/dashboard/festivals/add">
+          <Button>
+            <PlusIcon className="mr-2 h-4 w-4" />
+            Agregar Festival
+          </Button>
+        </Link>
```

Comment:
Acá podrías usar el componente `RedirectButton`

```suggestion
        <RedirectButton href="/dashboard/festivals/add">
          <PlusIcon className="mr-2 h-4 w-4" />
          Agregar Festival
        </RedirectButton>
```